tools/test-npm.sh

```diff
@@ -29,9 +29,10 @@ export npm_config_prefix="npm-prefix"
 export npm_config_tmp="npm-tmp"
 
 # install npm devDependencies and run npm's tests
-../$NODE_EXE cli.js install --ignore-scripts
-../$NODE_EXE cli.js run-script test-legacy
-../$NODE_EXE cli.js run-script test
+
+../$NODE cli.js install --ignore-scripts
+../$NODE cli.js run-script test-legacy
+../$NODE cli.js run-script test
```

Comment:
Actually on second thought this always used the local node to run cli.js anyways because these are relative links.

Still, I think this PR is better practice given our makefile setup.